applications/openshift/api-server/api_server_kubelet_https_serving_cert/rule.yml

```diff
@@ -0,0 +1,42 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Enable kubelet HTTPS connections to the API Server'
+
+description: |-
+    By default, the OpenShift API Server runs in HTTPS mode. HTTPS should be
+    used for connections between the API Server and Kubelets.
+    Edit the <tt>openshift-kube-apiserver</tt> configmap on the master node(s)
+    and remove the <tt>kubelet-https</tt> parameter if it is set to <tt>false</tt>:
+    <pre>
+    "apiServerArguments":{
+      ...
+      "kubelet-https":[
+        "false"
+      ],
+      ...
+    </pre>
+
+rationale: |-
+    Connections from the API Server to Kubelets could potentially carry
+    sensitive data such as secrets and keys. It is important to use
+    in-transit encryption for any communication between the API
+    Server and the Kubelets.
+
+severity: medium
+
+references:
+    cis: 1.2.4
+
+ocil_clause: |-
+    Verify that the serving-cert for the openshift-apiserver is type
+    <tt>kubernetes.io/tls</tt> and that returned Data includes <tt>tls.crt</tt>
+    and <tt>tls.key</tt>.
+
+ocil: |-
+    Run the following command on the master node(s):
+    <pre>$ oc -n openshift-apiserver describe secret serving-cert</pre>
+    Verify that the <tt>serving-cert</tt> for the openshift-apiserver is type
+    <tt>kubernetes.io/tls</tt> and that returned Data includes <tt>tls.crt</tt>
+    and <tt>tls.key</tt>.
```

Comment:
this one seems to be missing a rule to actually do the check. Is this intentional?

Reply:
This one wants to check a secret, so its just an informational rule to keep the operator permissions as they are.